src/parse/asp/interpreter_test.go

```diff
@@ -362,3 +362,9 @@ func TestFStringOptimisation(t *testing.T) {
 	assert.NotNil(t, assign.Optimised.Constant)
 	assert.EqualValues(t, "test", assign.Optimised.Constant)
 }
+
+func TestFormat(t *testing.T) {
+	s, err := parseFile("src/parse/asp/test_data/interpreter/format.build")
+	assert.NoError(t, err)
+	assert.EqualValues(t, `LLVM_NATIVE_ARCH=\"x86\"`, s.Lookup("arch"))
```

Comment:
Should `'{}_{}'.format('foo', 'bar')` produce `'foo_bar'`? It doesn't look like it will to me. Could we add a second `{}` to the test if ti should.